css/at-rules/font-face.json

```diff
@@ -258,7 +258,7 @@
           "__compat": {
             "description": "WOFF",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/CSS_fonts/WOFF",
-            "spec_url": "https://www.w3.org/TR/WOFF/#OverallStructure",
+            "spec_url": "https://www.w3.org/TR/WOFF/",
```

Comment:
This change just avoids the issue by not using a fragment.

The issue is that https://raw.githubusercontent.com/w3c/webref/main/ed/ids/WOFF.json only lists the fragments under https://w3c.github.io/woff/woff1/spec/Overview.html, although https://raw.githubusercontent.com/w3c/webref/main/ed/index.json does include https://www.w3.org/TR/WOFF/ as the version.